src/dlg_filter.h

```diff
@@ -48,17 +48,17 @@ typedef struct {
 //-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=--=-=-=-=
 // Class AuiNotebookNoKbd
 //-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=--=-=-=-=
-class AuiNotebookNoKbd : public wxAuiNotebook
+class AuiNotebookNoKbd : public wxNotebook
```

Comment:
Had to move from wxAuiNotebook to wxNotebook due to a stack overflow bug when creating instances of this object. Maybe wx 3.1.5 or later will resolve?